src/pillowtalk_impl.c

```diff
@@ -1040,3 +1080,5 @@ static void generate_node_json(pt_node_t* node, yajl_gen g)
     yajl_gen_null(g);
   }
 }
+
+/* vim: set tabstop=2 shiftwidth=2: */
```

Comment:
It seems that there is lots of coding conventions  from many communities...
And i just found this tool -- `clang-format`.
It provide some default configure for us:
```
  -style=<string>           - Coding style, currently supports:
                                LLVM, Google, Chromium, Mozilla, WebKit.
                              Use -style=file to load style configuration from
                              .clang-format file located in one of the parent
                              directories of the source file (or current
                              directory for stdin).
                              Use -style="{key: value, ...}" to set specific
                              parameters, e.g.:
                                -style="{BasedOnStyle: llvm, IndentWidth: 8}"
```
Maybe this is a good start point.
Which style would you vote for?